generators/kubernetes/templates/istio/destination-rule.yml.ejs

```diff
@@ -0,0 +1,27 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: <%= app.baseName.toLowerCase() %>-destinationrule
+  namespace: <%= kubernetesNamespace %>
+spec:
+  host: <%= app.baseName.toLowerCase() %>
+  trafficPolicy:
+    loadBalancer:
+      simple: RANDOM
+    connectionPool:
+      tcp:
+        maxConnections: 30
+        connectTimeout: 100ms
+      http:
+        http1MaxPendingRequests: 10
+        http2MaxRequests: 100
+        maxRequestsPerConnection: 10
+        maxRetries: 5
+    outlierDetection:
+      consecutiveErrors: 5
```

Comment:
@sendilkumarn I kept it with purpose. It's one way to let people know that these are the options available, so that post the generation if anything needs to be customised they can do so. If not, every time they have to refer the protobuf ref in the documentation